--- GPaste-3.40.2/meson.build~	2021-04-30 11:18:20.000000000 +0000
+++ GPaste-3.40.2/meson.build	2021-09-24 16:32:00.342777502 +0000
@@ -38,7 +38,7 @@
 libgpaste_deps = [ gdk_dep, gdk_pixbuf_dep, glib_dep, gtk_dep ]
 
 if get_option('introspection')
-  mutter_clutter_dep = dependency('mutter-clutter-8', version: mutter_clutter_req)
+  mutter_clutter_dep = dependency('mutter-clutter-9', version: mutter_clutter_req)
   gjs_dep = dependency('gjs-1.0', version: gjs_req)
 endif
 
